lib/pharos/host/ubuntu/scripts/install-kube-packages.sh

```diff
@@ -3,14 +3,14 @@
 set -e
 
 export DEBIAN_FRONTEND=noninteractive
-apt-mark unhold kubelet kubectl kubeadm || echo "Nothing to unhold"
-apt-get install -y "kubelet=${KUBE_VERSION}-00" "kubectl=${KUBE_VERSION}-00" "kubeadm=${KUBEADM_VERSION}-00"
-apt-mark hold kubelet kubectl kubeadm
+apt-mark unhold kubectl kubeadm kubernetes-cni || echo "Nothing to unhold"
```

Comment:
we don't need to hold and un-hold cni at this step, right?

Reply:
Good catch, fixed.